policy_machine.gemspec

```diff
@@ -23,5 +23,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency('database_cleaner')
   s.add_development_dependency('will_paginate', '~> 3.0.5')
   s.add_development_dependency('activerecord-hierarchical_query', '~> 0.0')
+  s.add_development_dependency('activerecord-import', '~> 0.0')
```

Comment:
Should we really not add this as a runtime dependency? I mean, if you decide to use another backend, you wont need it, but figuring out what dependencies a gem actually needs in an extra bundler fashion sounds slightly annoying.  Maybe a doc update? Oh, ditto for hierarchical_query above this as well.